lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -57,7 +57,7 @@ defmodule ExUnit.CaptureLog do
 
   It is possible to configure the level to capture with `:level`,
   which will set the capturing level for the duration of the
-  capture, for instance, if the log level is set to :error
+  capture, for instance, if the log level is set to `:error`
```

Comment:
Let's fix this too while we're at it:

```suggestion
  capture, for instance, if the log level is set to `:error`, then
```